src/dhcp_fsm.erl

```diff
@@ -156,10 +156,10 @@ offered(Pkg = #dhcp_package{xid = _XId, message_type = request},
             case delegate(request, Pkg, State) of
                 {ok, RPkg, State1} ->
                     Timeout = dhcp_package:get_option(ip_address_lease_time, dhcp_package:get_option(ip_address_lease_time, 3000, RPkg), Pkg),
```

Comment:
Line 158 is too long:                     Timeout = dhcp_package:get_option(ip_address_lease_time, dhcp_package:get_option(ip_address_lease_time, 3000, RPkg), Pkg),.